apps/oozie/src/oozie/views/dashboard.py

```diff
@@ -1193,7 +1193,17 @@ def check_job_edition_permission(oozie_job, user):
 
 
 def has_job_edition_permission(oozie_job, user):
-  return user.is_superuser or oozie_job.user == user.username or (oozie_job.group and user.groups.filter(name=oozie_job.group).exists()) or (oozie_job.acl and user.username in oozie_job.acl.split(','))
+  have_permission = user.is_superuser or oozie_job.user == user.username or (oozie_job.acl and user.username in oozie_job.acl.split(','))
```

Comment:
Why do you need to check ```oozie_job.acl``` in addition to ```user.username in oozie_job.acl.split(',')```?
Won't ```oozie_job.acl``` be the same as ```user.username in oozie_job.acl.split(',')``` (since for ```user.username in oozie_job.acl.split(',')``` to be True ```oozie_job.acl``` must have content (therefore is also True) and if ```user.username in oozie_job.acl.split(',')``` is False the ```oozie_job.acl``` won't matter since it's an And clauses)  in any case makeing the ```oozie_job.acl``` redundant?

Reply:
This is the original code

Reply:
You can still fix it in your commit also😄

Reply:
oozie_job.acl happens to be None in certain case, hence preventing an exception

Reply:
Would modifying
user.groups.filter(name=oozie_job.group).exists())
-->
user.groups.filter(name__in=oozie_job.group.split(",")).exists())
be enough?